src/tools/Citations.cpp

```diff
@@ -26,14 +26,23 @@
 
 namespace PLMD {
 
+const static Tools::FastStringUnorderedMap<std::string> doi_map = {
```

Comment:
@gtribello can you double check if makefile automatically detects this as a dependency? I mean, in this directory:
- `make obj`
- `touch Citations.cpp`
- `make obj` # here Citations.o should be recompiled

Reply:
(if not we can add it manually to the Makefile)

Reply:
I tried this and observed the following:

````
(plumed-cf) garethtribello@Gareths-MacBook-Pro tools % make obj                
compiling  BitmaskEnum.cpp
compiling  Citations.cpp
compiling  DLLoader.cpp
compiling  FileBase.cpp
compiling  HistogramBead.cpp
compiling  IFile.cpp
compiling  Keywords.cpp
compiling  LeptonCall.cpp
compiling  LinkCells.cpp
compiling  OFile.cpp
compiling  PDB.cpp
compiling  PlumedHandle.cpp
compiling  SwitchingFunction.cpp
compiling  Tree.cpp
(plumed-cf) garethtribello@Gareths-MacBook-Pro tools % touch Citations.cpp     
(plumed-cf) garethtribello@Gareths-MacBook-Pro tools % make obj           
compiling  Citations.cpp
````

I assume that means we are all good?